libmscore/edit.cpp

```diff
@@ -4664,8 +4672,8 @@ void Score::undoAddElement(Element* element)
                         if (ne->isHarmony()) {
                               for (Element* segel : segment->annotations()) {
                                     if (segel && segel->isFretDiagram() && segel->track() == ntrack) {
-                                          ne->setTrack(segel->track());
-                                          ne->setParent(segel);
+                                          FretDiagram* fd = toFretDiagram(segel);
```

Comment:
fd is not used. Is this a typo, or maybe the following line was meant to be "fd->add(ne)"?

Reply:
and `segEl` might be a better name for that variable

Reply:
ah yes, this line is unnecessary.